File: java17-junit5/src/test/java/com/katas/GreeterTest.java
package com.katas;

import org.junit.jupiter.api.Test;
import static org.assertj.core.api.Assertions.assertThat;

public class GreeterTest {

    @Test
    public void should_greet_helloworld() {
        Greeter greeter = new Greeter();

        String greeting = greeter.greet();

        assertThat(greeting).isEqualTo("Hello World!");
    }
}

File: kotlin-junit5/src/test/kotlin/com/katas/GreeterTest.kt
package com.katas

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class GreeterTest {
    @Test
    fun `should say hello`() {
        val greeter = Greeter("John Doe")

        assertThat(greeter.sayHello()).isEqualTo("Hello, John Doe!")
    }
}
